webpack.config.js

```diff
@@ -55,6 +55,17 @@ module.exports = env => ({
             }
           }
         ]
+      },
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          // Creates `style` nodes from JS strings
+          "style-loader",
```

Comment:
Should we use this or an external CSS file?

Reply:
I don't know. Is the common convention to use a file instead?

Reply:
What I meant was:

With the `style-loader` we put the CSS into a `<style>` tag. The CSS itself is all transported via JS. If we'd use the CSS extraction plugin we'd get a dedicated file and the loader would need to create a `<link>` instead.

Normally, I'd prefer the latter as it gives better performance in a number of scenarios (caching, discoverability, ...).

Reply:
(and yes, I'd say the standard is to use CSS in a `<link>` / external file instead of `<style>`; similar to JS which is usually used in dedicated files and not inline)

Reply:
@denniskigen Do you know how to move forward with this? @FlorianRappl makes a sound argument.

Reply:
I don't. Would you mind expounding on exactly what would need to be done here @FlorianRappl?

Reply:
Yeah we would need to change the `style-loader` and use, e.g., the `mini-css-extract-plugin` instead. This way, we get a `.css` file instead of a `<style>` tag. The plugin comes with a loader, which should place the `<link>` instead.